### PR TITLE
Cc/comb docs fix

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
A lot of my changes will no longer apply without a lot of diff analysis. I'm doing some documentation work over the next week or so anyway, so will just resubmit.

This is what survived.

### CODE REVIEW

**Comment by ChrisChinchilla** [comment]
I think this has squashed @mfussenegger but I must admit I'm a tad confused by this one :/

**Comment by mfussenegger** [comment]
Can you keep the lines wrapped at 80 chars? Otherwise ack

**Comment by ChrisChinchilla** [comment]
Odd @mfussenegger I have a code wrap set in my editor. I will check.

**Comment by ChrisChinchilla** [comment]
Yes I do @mfussenegger Can you show me an example where you think / are sure it's over 80 chars?

**Comment by ChrisChinchilla** [comment]
Try now @mfussenegger

File: docs/integration/squirrel.txt [hunk 1]
```diff
@@ -64,4 +63,5 @@ and scripts.
 
 .. _SQuirreL: http://www.squirrelsql.org
 .. _Crate JDBC: https://crate.io/docs/projects/crate-jdbc/stable/
-.. _Crate JDBC standalone jar: https://bintray.com/crate/crate/crate-jdbc/view/files/io/crate/crate-jdbc-standalone
\ No newline at end of file
+.. _Crate JDBC standalone jar:
```
Comment (mfussenegger) [inline docs/integration/squirrel.txt]: does this work correctly with sphinx if there are line breaks here? (is the doc still rendered correctly?)

Comment (mfussenegger) [inline docs/integration/squirrel.txt]: just checked it, doesn't render correctly: 
screenshot

**Comment by ChrisChinchilla** [comment]
I think this is resolved @mfussenegger I guess that's the issue with an automated line breaker, it doesn't appreciate subtlety… Though of course it now means you also have a line that's longer than 80 characters, I assume that's what you want.

**Comment by mfussenegger** [comment]
yep that's okay.

ack

**Comment by ChrisChinchilla** [comment]
Squashed @mfussenegger

**Pull request merged**